vaadin-spring-tests/test-spring-common/src/test/java/com/vaadin/flow/spring/test/routescope/BeansWithNoOWnerIT.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2000-2021 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.spring.test.routescope;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import com.vaadin.flow.spring.test.AbstractSpringTest;
+
+public class BeansWithNoOWnerIT extends AbstractSpringTest {
+
+    @Override
+    protected String getTestPath() {
+        return "/beans-no-owner";
+    }
+
+    @Test
+    public void beansWithNoOwner_preservedWithinTheSameRouteTarget_notPreservedAfterNavigation() {
+        open();
+
+        WebElement button = findElement(By.id("no-owner-button"));
+        String buttonId = button.getText();
+
+        switchContent();
+
+        WebElement div = findElement(By.id("no-owner-div"));
+        String divId = div.getText();
```

Comment:
Non-blocking: it is slightly confusing to name this as _id_ as it as a different meaning in this context (element id), using _uuid_ would have been clearer.

Reply:
It's an id for the div abstract component.
There is an element `id` , yes but it's not used anyhow here.
It's a bit confusing, yes, but I can't find a better name here.